app/lib/frontend/templates/views/pkg/admin_page.mustache

```diff
@@ -48,32 +48,52 @@
   {{/user_has_publisher}}
 </div>
 
-<h2>Discontinued</h2>
-{{^is_discontinued}}
+<h2>Package Options</h2>
+
+<h3>Discontinued</h3>
 <div>
-  <p>
-    If no one is maintaining this package, you can mark it as discontinued.
-    Discontinued packages remain available, but pub.dev doesn’t display them in search results.
-  </p>
-  <p>
-    <button
-      id="-admin-is-discontinued-toggle"
-      class="mdc-button mdc-button--raised"
-      data-mdc-auto-init="MDCRipple">Mark as "discontinued"</button>
-  </p>
+  If no one is maintaining this package, you can mark it as discontinued.
+  Discontinued packages remain available, but pub.dev doesn’t display them
+  in search results and won't analyze it for scoring.
```

Comment:
Maybe something like:
```html
A package can be marked as <a href="https://dart.dev/tools/pub/publishing#discontinue">discontinued</a>
to inform users that the package is no longer maintained.
<i>Discontinued packages</i> remain available to package users, but they will not show up
in search results on pub.dev unless explicitly included in the advanced search options.
```

@kwalrath if you have any quick suggestions for explanatory text feel free to leave us some wisdom :D

This is the "bla bla bla" section from the mockup in go/is:unlisted-on-pub